crates/chainio/clients/elcontracts/src/reader.rs

```diff
@@ -12,7 +12,7 @@ use eigen_utils::middleware::{
 pub struct ELChainReader {
     _logger: SharedLogger,
     slasher: Address,
-    delegation_manager: Address,
+    pub delegation_manager: Address,
```

Comment:
Can we reduce this to `pub(crate)`? That way external users cannot access it.

Reply:
done